### PR TITLE
logalize 0.4.1 (new formula)

### PR DESCRIPTION
**Pull request by deponian**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/l/logalize.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Logalize < Formula
+  desc "Fast and extensible log colorizer. Alternative to ccze"
+  homepage "https://github.com/deponian/logalize"
+  url "https://github.com/deponian/logalize.git",
+    tag:      "v0.4.1",
+    revision: "2d74f36f36037d1a09371f044a85b2b8f192905f"
```
Comment (chenrui333) [inline Formula/l/logalize.rb:6]:
```suggestion
  url "https://github.com/deponian/logalize/archive/refs/tags/v0.4.1.tar.gz"
  sha256 "xxx"
```

File: Formula/l/logalize.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    system "make", "VERSION=#{version}", "build"
+    bin.install "dist/#{version}/logalize"
+
+    system "make", "VERSION=#{version}", "manpage"
+    man1.install "manpages/logalize.1.gz"
+
+    system "make", "completions"
+    bash_completion.install "completions/logalize.bash" => "logalize"
+    zsh_completion.install "completions/logalize.zsh" => "_logalize"
+    fish_completion.install "completions/logalize.fish"
```
Comment (chenrui333) [inline Formula/l/logalize.rb:22]:
```suggestion
    ldflags = "xxx"
    system "go", "build", *std_go_args(ldflags:), "./cmd"

    generate_completions_from_executable(bin/"logalize", "completion")
```

File: Formula/l/logalize.rb (hunk 1 above)
Comment (deponian) [reply Formula/l/logalize.rb:6]: The application needs commit hash during the building stage, so I need to use tag-revision approach here

File: Formula/l/logalize.rb (hunk 2 above)
Comment (deponian) [reply Formula/l/logalize.rb:22]: I's much simpler for me as developer of the application to build everything using `make`. This way I'll be able to change building process in the future without changing the formula

**Comment by deponian** [comment]
@chenrui333 Hi! So there is no way to add Logalize to Homebrew Core without 75 starts on Github, right?

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by SMillerDev** [comment]
Please also note:
> We frown on authors submitting their own work unless it is very popular.

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff:~:text=We%20frown%20on%20authors%20submitting%20their%20own%20work%20unless%20it%20is%20very%20popular.

**Pull request closed without merging**